### PR TITLE
Map Component

### PR DESCRIPTION
**Pull request by DieS06**
Map component created and displayed in the dashboard.

### CODE REVIEW

**Review by ichivancrCenfotec — APPROVED** [review]

File: src/app/components/locationGMaps/locationGMaps.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    fixture = TestBed.createComponent(LocationGMapsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create the component', () => {
+    expect(component).toBeTruthy();
+  });
+
+//   it('should initialize the map', () => {
```
Comment (nefi-cenfo) [inline src/app/components/locationGMaps/locationGMaps.component.spec.ts:25]: Is this commented code needed?

File: src/environments/environment.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
+
 export const environment = {
-  production: true,
-  apiUrl: 'http://localhost:8080'
+    production: false,
+    apiUrl: "undefined",
```
Comment (nefi-cenfo) [inline src/environments/environment.ts:4]: Please check why the values are undefined.

File: src/index.html [hunk 3]
```diff
@@ -10,12 +10,13 @@
...
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-  <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script src="https://maps.googleapis.com/maps/api/js?key="{{gMapsKey}} type="text/javascript"></script>
```
Comment (nefi-cenfo) [inline src/index.html:18]: Please add the 'defer' attribute to the tag.

**Review by nefi-cenfo — APPROVED** [review]

File: src/index.html (hunk 3 above)
Comment (DieS06) [reply src/index.html:18]: What is the defer attribute?

**Pull request closed without merging**